### PR TITLE
Fix nb package version

### PR DESCRIPTION
**Pull request by maroroman**
- [x] The commits are squashed in a cohesive manner and have meaningful messages.
- [x] For commits that came from upstream, `[UPSTREAM]` has been prepended to the commit message
- [x] JIRA link(s): https://issues.redhat.com/browse/RHODS-2425
- [x] The Jira story is acked
- [x] An entry has been added to the latest build document in [Build Announcements Folder](https://drive.google.com/drive/folders/1sgkK1WZgGo9CXsLizNe0GbAzVKuSKrZL).
- [x] Testing instructions have been added in the PR body (for PRs involving changes that are not immediately obvious)

### CODE REVIEW

File: jupyterhub/notebook-images/overlays/additional/minimal-notebook-imagestream.yaml [hunk 1]
```diff
@@ -16,7 +16,7 @@ spec:
   tags:
   - annotations:
       opendatahub.io/notebook-software: '[{"name":"Python","version":"v3.8"}]'
-      opendatahub.io/notebook-python-dependencies: '[{"name":"JupyterLab","version": "3.2.4"}, {"name": "Notebook","version": "6.4"}]'
+      opendatahub.io/notebook-python-dependencies: '[{"name":"JupyterLab","version": "3.2"}, {"name": "Notebook","version": "6.0"}]'
```
Comment (lugi0) [inline jupyterhub/notebook-images/overlays/additional/minimal-notebook-imagestream.yaml:19]: Notebook should still be `6.4` rather than `6.0`

**Review by lugi0 — CHANGES_REQUESTED** [review]

**Review by crobby — CHANGES_REQUESTED** [review]
As luca said, notebook should be 6.4 rather than 6.0

**Review by lugi0 — APPROVED** [review]
lgtm

**Comment by samuelvl** [comment]
lgtm

**Pull request merged**